src/cli-options.ts

```diff
@@ -222,6 +223,22 @@ export const getOptions = (): Result<Options> => {
     return loggerResult;
   }
 
+  let logger = loggerResult.value;
+  if (process.env['WIREIT_DEBUG_LOG_TO']) {
```

Comment:
I think I slightly prefer `WIREIT_DEBUG_LOG_FILE` over `WIREIT_DEBUG_LOG_TO`

Reply:
I like that too, renamed